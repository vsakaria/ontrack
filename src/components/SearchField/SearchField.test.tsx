import React from "react";
import { cleanup, render } from "@testing-library/react";
import { SearchField } from ".";

afterAll(cleanup);

test("SearchField should render the correct DOM", () => {
  const { container } = render(<SearchField handleSubmit={jest.fn()} />);
  expect(container).toMatchSnapshot("SearchField");
});
