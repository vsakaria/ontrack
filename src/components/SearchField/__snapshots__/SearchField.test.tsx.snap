// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchField should render the correct DOM: SearchField 1`] = `
<div>
  <form
    class="mb-sm-3 mt-sm-3"
    data-testid="form"
  >
    <div
      class="input-group"
    >
      <input
        class="form-control"
        placeholder="Search"
        type="text"
      />
      <div
        class="input-group-prepend"
      >
        <button
          class="btn btn-primary"
          type="submit"
        >
          Search
        </button>
      </div>
    </div>
  </form>
</div>
`;
